OpenEdXMobile/src/main/java/org/edx/mobile/player/VideoPlayer.java

```diff
@@ -542,6 +540,9 @@ public synchronized void start() throws IllegalStateException {
                 || state == PlayerState.STOPPED
                 || state == PlayerState.LAGGING
                 || state == PlayerState.PLAYBACK_COMPLETE) {
+            if (seekToWhenPrepared > 0) {
+                seekTo(seekToWhenPrepared);
+            }
```

Comment:
Why was this code block moved here?

Reply:
@miankhalid This block of code moved here to resolve the issue of restarting the player(from the last saved position) after completed when playback speed is more then 1.0x.